my-app/src/App.js

```diff
@@ -24,33 +21,14 @@ function App() {
   };
 
   useEffect(() => {
-    // Save the user data to storage whenever the user state changes
     localStorage.setItem("user", JSON.stringify(user));
   }, [user]);
 
   const [watchedVideosCount, setWatchedVideosCount] = useState(0);
-  const [ttvideos, setTTVideos] = useState([]);
-
-  useEffect(() => {
```

Comment:
What is this code you took away? I can't tell what it was doing before

Reply:
Yeah it wasn't doing anything there when I took a clear look at it . I think I was trying to implement a feature and forgot to take out that code.

Reply:
Sounds good!